tsconfig.app.json

```diff
@@ -8,7 +8,6 @@
   "include": ["src/**/*.d.ts"],
   "exclude": ["src/test.ts", "src/**/*.spec.ts"],
   "angularCompilerOptions": {
-    "preserveWhitespaces": true,
-    "enableIvy": false
```

Comment:
`enableIvy` is obsolete in Angular `v12`. Having this property results in a warning when compiling.